.github/workflows/acceptance_tests_buildin.yaml

```diff
@@ -0,0 +1,204 @@
+name: Acceptance Tests (Buildin)
+on:
+  pull_request:
+    types: [ opened, labeled, synchronize ]
+env:
+  MAVEN_VERSION: 3.6.3
+  MAVEN_SHA: c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
+  BUILDIN: true
+  DEPLOYMENT_NAME: "autoscaler-${{ github.event.pull_request.number }}-buildin"
```

Comment:
To keep in line with the other deployments it should be starting with app-autoscaler.